x-pack/test/functional/apps/ml/anomaly_detection/lens_to_ml.ts

```diff
@@ -0,0 +1,222 @@
+    await PageObjects.dashboard.loadSavedDashboard(dashboardTitle);
+    await ml.dashboardEmbeddables.assertDashboardPanelExists(selectedPanelTitle);
+
+    await setFarequoteTimerange();
+
+    const header = await dashboardPanelActions.getPanelHeading(selectedPanelTitle);
+    await dashboardPanelActions.openContextMenuMorePanel(header);
+  }
+
+  async function createJobInWizard(
+    jobId: string,
+    splitField?: string,
+    aggAndFieldIdentifier?: string
+  ) {
+    await headerPage.waitUntilLoadingHasFinished();
+
+    if (splitField !== undefined) {
+      await ml.jobWizardMultiMetric.assertDetectorSplitExists(splitField);
+      await ml.jobWizardCommon.assertInfluencerSelection([splitField]);
+    }
+
+    if (aggAndFieldIdentifier !== undefined) {
+      await ml.jobWizardCommon.assertAggAndFieldInputExists();
+      await ml.jobWizardCommon.selectAggAndField(aggAndFieldIdentifier, true);
+      await ml.jobWizardCommon.assertAnomalyChartExists('LINE');
+    }
+
+    await ml.testExecution.logTestStep('job creation displays the job details step');
+    await ml.jobWizardCommon.advanceToJobDetailsSection();
+
+    await ml.testExecution.logTestStep('job creation inputs the job id');
+    await ml.jobWizardCommon.assertJobIdInputExists();
+    await ml.jobWizardCommon.setJobId(jobId);
+
+    await ml.testExecution.logTestStep('job creation displays the validation step');
+    await ml.jobWizardCommon.advanceToValidationSection();
+
+    await ml.testExecution.logTestStep('job creation displays the summary step');
+    await ml.jobWizardCommon.advanceToSummarySection();
+
+    await ml.testExecution.logTestStep('job creation creates the job and finishes processing');
+    await ml.jobWizardCommon.assertCreateJobButtonExists();
+    await ml.jobWizardCommon.createJobAndWaitForCompletion();
+
+    await ml.testExecution.logTestStep('job creation displays the created job in the job list');
+    await ml.navigation.navigateToMl();
+    await ml.navigation.navigateToJobManagement();
+
+    await ml.jobTable.filterWithSearchString(jobId, 1);
+
+    await ml.jobTable.assertJobRowJobId(jobId);
+  }
+
+  describe('anomaly charts in dashboard', function () {
+    this.tags(['ml']);
+
+    before(async () => {
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/apps/ml/anomaly_detection/fixtures/lens_to_ml.json'
+      );
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+      await kibanaServer.importExport.unload(
+        'x-pack/test/functional/apps/ml/anomaly_detection/fixtures/lens_to_ml.json'
+      );
+    });
+
+    beforeEach(async () => {
+      await PageObjects.common.navigateToApp('dashboard');
+    });
+
+    describe('tests create job from lens', () => {
```

Comment:
I don't think that we need to sub-suite level. The `afterEach` could just move up and all the tests could be run on the top level suite of this test file.

Reply:
https://github.com/elastic/kibana/pull/133899/commits/b05cf28b33b439400f16dcc749d72e7edac877f0